core/views.py

```diff
@@ -31,7 +31,15 @@ class UserRegistrationWizard(CookieWizardView):
     template_name = 'core/user_registration.html'
     
     def done(self, form_list, **kwargs):
-        return render_to_response('core/success.html')
+        """Save info from completed forms to database"""
+        # https://stackoverflow.com/questions/14791892/saving-django-formwizard
+        data = {}
+        for form in form_list:
+            data.update(form.cleaned_data)
+        
+        User.objects.create(**data)
```

Comment:
The way to create user is ``User.objects.create_user(**data)``
Otherwise, password will not be hashed